dockerfiles/nodemon.json

```diff
@@ -2,7 +2,7 @@
     "verbose": false,
     "delay": 2000,
     "ext": "py",
-    "watch": ["readthedocs", "readthedocsinc"],
+    "watch": ["readthedocs", "readthedocsinc", "../readthedocs.org/readthedocs"],
```

Comment:
Does this work properly on .org? Doesn't it fails saying this directory doesn't exists there? Should we remove the first `"readthedocs"` to avoid watching the same files twice?

Reply:
It's in the `common` repo, so it'd have to work on both repos at the same time.

But readthedocs.org never failed with the missing `readthedocsinc` path spec, so it's safe :+1:

Reply:
Works on .org :+1: